collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderActivity.java

```diff
@@ -237,18 +256,21 @@ public void uploadingComplete(HashMap<String, String> result) {
                                 results.getString(
                                         results.getColumnIndex(InstanceColumns.DISPLAY_NAME));
                         String id = results.getString(results.getColumnIndex(InstanceColumns._ID));
-                        message.append(name + " - " + result.get(id) + "\n\n");
+                        queryMessage.append(name + "-" + result.get(id) + "\n\n");
                     }
-                } else {
-                    message.append(getString(R.string.no_forms_uploaded));
                 }
+            } catch (SQLException e) {
+                Log.e(TAG, e.getMessage());
             } finally {
-                if (results != null) {
+                if (results != null)
                     results.close();
-                }
             }
+            message.append(queryMessage.toString());
         }
-
+        if (message.length() == 0) {
+            message.append(getString(R.string.no_forms_uploaded));
+        }
+        Log.d(TAG,message.toString());
```

Comment:
Ohh sorry i forgot to remove this it was for debugging purpose. And yes there must be some space between params.